web/ui-app-v2/src/screens/Feedback/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import RatingsComponent from "./components/Ratings";
+import ButtonGroupComponent from "./components/ButtonToggle";
+import TextAreaComponent from "./components/TextArea";
+import RaisedButton from "material-ui/RaisedButton";
+import "./index.css";
+import cloneDeep from "lodash/cloneDeep";
+
+class Feedback extends Component {
+  state = {
+    items: [
+      {
+        label: "Service",
+        style: {
+          border: "1px solid #f5a623",
+          borderRadius: "3px",
+          marginRight: "5px",
+          background: "transparent",
+          height: "auto",
+          lineHeight: "30px",
+        },
+        labelStyle: {
+          textTransform: "none",
+          fontWeight: "900",
+          color: "#484848",
+        },
+      },
+      {
+        label: "Resolution Time",
+        style: {
+          border: "1px solid #f5a623",
+          borderRadius: "3px",
+          marginRight: "5px",
+          background: "transparent",
+          height: "auto",
+          lineHeight: "30px",
+        },
+        labelStyle: {
+          textTransform: "none",
+          fontWeight: "900",
+          color: "#484848",
+        },
+      },
+      {
+        label: "Other",
+        style: {
+          border: "1px solid #f5a623",
+          borderRadius: "3px",
+          marginRight: "5px",
+          background: "transparent",
+          height: "auto",
+          lineHeight: "30px",
+        },
+        labelStyle: {
+          textTransform: "none",
+          fontWeight: "900",
+          color: "#484848",
+        },
+      },
+    ],
+  };
```

Comment:
I see a lot of  borderRadius: "3px",
+          ```
marginRight: "5px",
+          background: "transparent",
+          height: "auto",
+          lineHeight: "30px",
```. Can we not include it via css?